### PR TITLE
#2 - Create product-detail

### PR DESCRIPTION
**Pull request by tuannt02**
- Icon component
- Banner component
- FeatureItem component
- Configure h1,..h6 tags
- Vertical Scroll (overlay)
- A little bit more ...

### CODE REVIEW

File: src/styles/index.css [hunk 1]
```diff
@@ -21,6 +21,38 @@
 }
 
 body {
+  /* Scrollbar vertical */
+  overflow-y: overlay;
+
```
Comment (vuhuy172dh) [inline src/styles/index.css:26]: unnecessary. It should be removed.

File: src/components/featureItem.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <div className="grid grid-cols-4 gap-x-5 laptop:grid-cols-12 laptop:gap-y-5">
+        {featureList.map((item) => {
+          return (
+            <div
+              key={item.label}
+              className="px-6 py-9 bg-light_grey mt-6 col-span-4 laptop:col-span-3 laptop:mt-0"
+            >
+              <Icon icon={item.icon} />
+              <h4 className="text-h4 mt-4 mb-3">{item.label}</h4>
+              <p className="text-h6 font-satoshi">{item.desc}</p>
```
Comment (vuhuy172dh) [inline src/components/featureItem.jsx]: replace "text-h6 font-satoshi" with "text-body-sm".

File: src/pages/productDetail.jsx [hunk 3]
```diff
@@ -1,5 +1,124 @@
+import Button from '../components/button'
+import Banner from '../components/banner'
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: remove Banner

File: src/pages/productDetail.jsx [hunk 4]
```diff
@@ -1,5 +1,124 @@
...
+// Sample img
+import DandyChairImg from '../assets/images/DandyChair.png'
+import WhiteRoomImg from '../assets/images/features3.png'
+
 function ProductDetailPage() {
-  return <div>This is Product Detail Page</div>
+  return (
+    <div className="text-dark_primary mx-6 laptop:mx-20">
+      {/* Banner */}
+      <Banner />
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: remove!!!

File: src/pages/productDetail.jsx [hunk 5]
```diff
@@ -1,5 +1,124 @@
...
+            <h3 className="text-h3 mb-3 laptop:text-h1 laptop:mb-4">
+              The Dandy Chair
+            </h3>
+            <h4 className="text-h4 laptop:text-h3">£250</h4>
+          </div>
+          <hr />
+          {/* Description */}
+          <div className="mt-7 laptop:mt-6">
+            <h5 className="text-h5 mb-3">Product description</h5>
+            <small className="text-h6 font-satoshi laptop:text-h5">
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: replace "text-h6 font-satoshi laptop:text-h5" with "text-body-sm laptop:text-body-md"

File: src/pages/productDetail.jsx [hunk 6]
```diff
@@ -1,5 +1,124 @@
...
+          </div>
+          {/* Dimensions */}
+          <div className="mt-7 laptop:mt-9 laptop:w-80">
+            <h5 className="text-h5 mb-3 laptop:mb-6">Dimensions</h5>
+            <hr className="hidden" />
+            {/* Dimensions Metrics */}
+            <div className="flex justify-between items-center mt-4 ">
+              <div>
+                <h6 className="text-h6 mb-4">Height</h6>
+                <small className="text-h6 font-satoshi laptop:text-h5">
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: replace "text-h6 font-satoshi laptop:text-h5" with "text-body-sm laptop:text-body-md"

File: src/pages/productDetail.jsx [hunk 7]
```diff
@@ -1,5 +1,124 @@
...
+                  110cm
+                </small>
+              </div>
+              <div className="h-11 w-[1px] bg-gray-300"></div>
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: replace "w-[1px] bg-gray-300" with "w-1 bg-border_grey"

File: src/pages/productDetail.jsx [hunk 8]
```diff
@@ -1,5 +1,124 @@
...
+              <div>
+                <h6 className="text-h6 mb-4">Width</h6>
+                <small className="text-h6 font-satoshi laptop:text-h5">
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: replace "text-h6 font-satoshi laptop:text-h5" with "text-body-sm laptop:text-body-md"

File: src/pages/productDetail.jsx [hunk 9]
```diff
@@ -1,5 +1,124 @@
...
+                  75cm
+                </small>
+              </div>
+              <div className="h-11 w-[1px] bg-gray-300"></div>
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: replace "w-[1px] bg-gray-300" with "w-1 bg-border_grey"

File: src/pages/productDetail.jsx [hunk 10]
```diff
@@ -1,5 +1,124 @@
...
+              <div>
+                <h6 className="text-h6 mb-4">Depth</h6>
+                <small className="text-h6 font-satoshi laptop:text-h5">
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: replace "text-h6 font-satoshi laptop:text-h5" with "text-body-sm laptop:text-body-md"

File: src/pages/productDetail.jsx [hunk 11]
```diff
@@ -1,5 +1,124 @@
...
+      <section className="flex flex-col laptop:flex-row laptop:-mx-20">
+        <div className="-mx-6 laptop:basis-3/6 laptop:mx-0">
+          <img className="w-full h-auto" src={WhiteRoomImg} alt="feature" />
+        </div>
+        <div className="flex flex-col py-9 laptop:p-10 laptop:basis-3/6">
+          <div className="mb-14 text-h4">
+            <h4 className="text-h4 mb-4 laptop:text-h2 laptop:mb-5">
+              Join the club and get the benefits
+            </h4>
+            <small className="text-h6 font-satoshi laptop:text-h5">
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: replace "text-h6 font-satoshi laptop:text-h5" with "text-body-sm laptop:text-body-md"

File: src/pages/productDetail.jsx [hunk 12]
```diff
@@ -1,5 +1,124 @@
...
+          <div className="flex-1"></div>
+          <div className="flex laptop:max-w-md">
+            <input
+              className="flex-1 overflow-auto px-8 bg-light_grey"
+              type="text"
+              placeholder="your@email.com"
+            />
+            <Button className="grow-0" Size="medium" Color="primary">
+              Sign up
+            </Button>
```
Comment (vuhuy172dh) [inline src/pages/productDetail.jsx]: Replace this with EmailField component.

**Review by vuhuy172dh — COMMENTED** [review]
1. Remove banner.jsx file
2. Switch the content of featureItem.jsx to infoItem.jsx. I have created a file for you but you don't use it. 
3. You create a folder constants and move constant.js file to this folder and rename it to iconConstant.js
4. You should create a folder helper and move icon.jsx file to this folder. Should be icon.js (not icon.jsx).

File: src/styles/index.css (hunk 1 above)
Comment (tuannt02) [reply src/styles/index.css:26]: Agree to keep. No change

File: src/components/featureItem.jsx (hunk 2 above)
Comment (tuannt02) [reply src/components/featureItem.jsx]: ok

File: src/pages/productDetail.jsx (hunk 3 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: ok

File: src/pages/productDetail.jsx (hunk 4 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: ok

File: src/pages/productDetail.jsx (hunk 5 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: ok

File: src/pages/productDetail.jsx (hunk 8 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: ok

File: src/pages/productDetail.jsx (hunk 10 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: ok

File: src/pages/productDetail.jsx (hunk 9 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: ok

File: src/pages/productDetail.jsx (hunk 11 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: ok

File: src/pages/productDetail.jsx (hunk 12 above)
Comment (tuannt02) [reply src/pages/productDetail.jsx]: Maybe problem with EmailField component.

**Comment by vuhuy172dh** [comment]
Merged!!!!!

**Pull request merged**